tools/highlighter-helper/src/client/styles/line-numbers.scss

```diff
@@ -23,7 +23,7 @@ div[class*='language-'] {
       border-right: 1px solid var(--code-c-highlight-bg, var(--code-c-text));
       border-radius: var(--code-border-radius) 0 0 var(--code-border-radius);
 
-      transition: border var(--t-color);
+      transition: border var(--vp-t-color);
```

Comment:
nice catch, could this be in a sperate PR?

Reply:
I saw that it was fixed in another commit.